rfcs/text/0352-graph-type-system.md

```diff
@@ -2682,14 +2682,14 @@ A more immediately apparent concern is if schemas are versioned, but their use i
 
 ## The Big Picture
 
-As outlined in the [Main Motivation section](#main-motivation), there are some clear shortcomings to how the Graph service works at the moment in regards to how it expresses data structure. It's also important to mention that there is a tough balance between comprehensive functionality for:
+As outlined in the [Motivation section](#motivation), there are some clear shortcomings to how the Graph service works at the moment in regards to how it expresses data structure. It's also important to mention that there is a tough balance between comprehensive functionality for:
```

Comment:
@Alfred-Mountfield soes this change make sense to you? Spotted by a new default rule called [`link-fragments`](https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md#md051).

Reply:
I think it makes sense. I don't see a section called _Main_ motivation so it was likely a mistake. I thought we were ignoring RFCs for lints though to avoid introducing too much overhead for people to get them to pass, especially as they're just technical docs?

Reply:
+1 to this suggestion. I feel like we'd be better off linting RFCs ourselves (manually) _at the point of acceptance_ when we're ready to merge in to the repo, and it'd be preferable if lint tests weren't run (or more precisely weren't at risk of failing) by users simply iterating on drafts.

Reply:
We can think about CI triggers separately. AFAIR, dev.azure.com does not run CI for PR drafts but they can be triggered manually. Since Microsoft bought GitHub, they borrowed some useful features from dev.azure.com, so maybe there is an option to configure something similar. I would not focus on this right now since we don’t have external RFCs yet.

Reply:
I think it's more that we can just add RFCs to the ignore files for now

Reply:
Since we don't currently ignore RFCs, and linting them has actually spotted a broken link, I don't think we need to start ignoring them now.

People can ignore linting failures until they actually want to merge them, so I don't see it as introducing any real overhead.

Reply:
> CI can fail not only because of markdownlint, but prettier too. It needs to be also disabled for RFCs following the above logic

Well I do think there is some argument that we don't necessarily need `prettier` for RFCs either.

But I agree with this:
Since we don't currently ignore RFCs, and linting them has actually spotted a broken link, I don't think we need to start ignoring them now.